lib/rosette/integrations/smartling-integration/smartling_puller.rb

```diff
@@ -10,6 +10,12 @@ class SmartlingPuller
 
         DEFAULT_THREAD_POOL_SIZE = 10
 
+        PULL_STATUSES = [
```

Comment:
we may want to consider putting the status constants together in a single file for referencability